OGC-Core/OGCGeometry.class.st

```diff
@@ -1,43 +1,227 @@
-The instantiable subclasses of Geometry defined in this Standard are restricted to 0, 1 and
-2-dimensional geometric objects that exist in 2, 3 or 4-dimensional coordinate space. 
-Geometry values in R2 have points with coordinate values for x and y. Geometry values in R3 have points with coordinate
-values for x, y and z or for x, y and m. Geometry values in R4 have points with coordinate values for x, y, z and m.
-The interpretation of the coordinates is subject to the coordinate reference systems associated to the point. All
-coordinates within a geometry object should be in the same coordinate reference systems. Each coordinate shall
-be unambiguously associated to a coordinate reference system either directly or through its containing geometry.
-The z coordinate of a point is typically, but not necessarily, represents altitude or elevation. The m coordinate
-represents a measurement.
-All Geometry classes described in this standard are defined so that instances of Geometry are topologically
-closed, i.e. all represented geometries include their boundary as point sets. This does not affect their 
-"
-Class {
-	#name : #OGCGeometry,
-	#superclass : #Object,
-	#category : 'OGC-Core'
-}
-
-{ #category : #accessing }
-OGCGeometry >> dimension [ 
-	self subclassResponsibility 
-]
-
-{ #category : #accessing }
-OGCGeometry >> geometryType [
-	^ self class geometryType
-]
-
-{ #category : #accessing }
-OGCGeometry >> is3D [
-	self subclassResponsibility 
-]
-
-{ #category : #accessing }
-OGCGeometry >> isEmpty [ 
-	self subclassResponsibility 
-]
+"
+6.1.2 Geometry
+
+6.1.2.1 Description 
+
+Geometry is the root class of the hierarchy. Geometry is an abstract (non-instantiable) class.
+The instantiable subclasses of Geometry defined in this Standard are restricted to 0, 1 and
+2-dimensional geometric objects that exist in 2, 3 or 4-dimensional coordinate space. 
+Geometry values in R2 have points with coordinate values for x and y. Geometry values in R3 have points with coordinate
+values for x, y and z or for x, y and m. Geometry values in R4 have points with coordinate values for x, y, z and m.
+The interpretation of the coordinates is subject to the coordinate reference systems associated to the point. All
+coordinates within a geometry object should be in the same coordinate reference systems. Each coordinate shall
+be unambiguously associated to a coordinate reference system either directly or through its containing geometry.
+The z coordinate of a point is typically, but not necessarily, represents altitude or elevation. The m coordinate
+represents a measurement.
+All Geometry classes described in this standard are defined so that instances of Geometry are topologically
+closed, i.e. all represented geometries include their boundary as point sets. This does not affect their 
+"
+Class {
+	#name : #OGCGeometry,
+	#superclass : #Object,
+	#instVars : [
+		'spatialRS',

```

Comment:
We should avoid abbreviations on class or instance variable names. I know spatialReferenceSystem is long but it is understandable. Maybe spatial is in the context and referenceSystem is a good name

Reply:
However 2 reference systems exists: a measure reference system linked to m-value, and here, a spatial reference system for computations.
But if it's not confusing, we can consider referenceSystem = spatial reference system in an implicit way
+ Ok to write full name instead of RS, I'll change it!